app/domain/Capabilities.scala

```diff
@@ -43,17 +38,17 @@ object Capabilities {
   }
 
   case object ChangeClientSecret extends Capability {
-    def hasCapability(app: Application) = app.state.name == State.PRODUCTION
+    def hasCapability(app: Application) = app.state.name.isApproved
```

Comment:
Is it time to add helper methods to app so we can do
`app.isApproved`

and not care how this is done.

Likewise for other tests below.